app/controllers/places_controller.rb

```diff
@@ -1,21 +1,45 @@
 class PlacesController < ApplicationController
   PLACES_PER_PAGE = 9
 
+  before_action :logged_in_user, only: [:new, :create, :my_places, :destroy]
   before_action :place_find, only: [:show]
 
   def index
-    @places = Place.where(status: :created).paginate(page: params[:page], per_page: PLACES_PER_PAGE)
+    @places = Place.where(status: :created).paginate(page: params[:page],
+                                                     per_page: PLACES_PER_PAGE)
   end
 
-  def show; end
+  def show
+    place_find
+  end
+
+  def new
+    @place = Place.new
+    @address = Address.new
+    @picture = Picture.new
+  end
 
-  def new; end
+  def create
+    place_build && address_build && picture_build
+    if @place.save && @address.save && @picture.save
+      flash[:notice] = 'Place created'
+      redirect_to my_places_path
```

Comment:
why redirect to my places? i think, it'd be better to redirect to the place url instead:
```suggestion
      redirect_to place_path(@place)
```

Reply:
fixed)